src/paper/openalex_util.py

```diff
@@ -0,0 +1,146 @@
+import logging
```

Comment:
The code below is a refactored + optimized version of the code below:

https://github.com/ResearchHub/researchhub-backend/pull/1592/files#diff-28e810c7045544f9f4b257545fa75c388004e8b44fbbdb822e71eb8886860746L914

Reply:
It is 50% existing + 50% new I'd say